File: angular/src/app/expense/models/expense-dto.ts

import { ExpenseType } from './expense-type';

import { AuditedEntityDto } from '@abp/ng.core';

export class ExpenseDto extends AuditedEntityDto<string> {
  title: string;
  onDate: string;
  type: any;
  amount: number;
  lastModificationTime?: string;
  lastModifierId?: string;
  creationTime: string;
  creatorId?: string;
  id: string;

  constructor(initialValues: Partial<ExpenseDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/Nic.Octopus.Application.Contracts/ExpenseDto.cs
using System;
using Nic.Octopus.Domain.Shared;
using Volo.Abp.Application.Dtos;

namespace Nic.Octopus.Application.Contracts
{
    public class ExpenseDto : AuditedEntityDto<Guid>
    {
        public string Title { get; set; }

        public DateTime OnDate { get; set; }

        public ExpenseType Type { get; set; }

        public float Amount { get; set; }
    }
}